docs/REST API/rest-api.md

```diff
@@ -6953,7 +7015,7 @@ Deletes the session specified by ID. When the user associated with the session n
 
 #### Example
 
-`GET /api/v1/fleet/software`
+`GET /api/v1/fleet/software/versions`
```

Comment:
Because we're not documenting this, I think we should make it clear to the engineer building this feature that we're not breaking this endpoint. Instead we're giving it a second name `GET /software/versions`. 

Both `GET /software` and `GET /software/versions` will work the same. We're just document `GET /software/versions`

Can you please call this out in the PR description?

Furthermore, this reminded me that we're missing `GET /software/{id}` in the existing docs and in this PR. I think we do the same for this endpoint: add a second name `GET /software/versions/{id}` and only document this name.

What do you think?

Reply:
Furthermore, this reminded me that we're missing `GET /software/{id}` in the existing docs and in this PR. I think we do the same for this endpoint: add a second name `GET /software/versions/{id}` and only document this name.

What do you think?

Reply:
> Furthermore, this reminded me that we're missing GET /software/{id} in the existing docs and in this PR.
> I think we do the same for this endpoint: add a second name GET /software/versions/{id} and only document this name.

Agree.